external/iotivity/iotivity_1.2-rel/resource/csdk/stack/src/oickeepalive.c

```diff
@@ -274,7 +274,7 @@ OCStackResult OCFindKeepAliveResource(OCDoHandle *handle, const char *remoteAddr
 }
 
 OCStackResult OCSendKeepAliveRequest(OCDoHandle *handle, const char *remoteAddr,
-                                     OCPayload *payload, OCCallbackData *cbData)
+                                     OCRepPayload *payload, OCCallbackData *cbData)
```

Comment:
Instead of making changes in the prototype, typecast the parameter in the function call with (OCPayload *).

Reply:
ok